### ISSUE TITLE
Improve chemical dissolution

### ISSUE THREAD

**Issue by xyl96** [MindTheGap-ERC/CarboKitten.jl#66]
The reaction rate should be linked to: 1) temperature and 2) saturation index, instead of a fixed number right now. See Table 4 of [Kaufman et al., 2007](https://www.sciencedirect.com/science/article/pii/S001670370602240X) as an example. 
Quantifying the reaction rate can use [Transient State Theory (TST)](https://www.sciencedirect.com/topics/chemistry/transition-state-theory#:~:text=Transition%20state%20theory%20(TST)%20provides,Polanyi%20%5B104%5D%20in%201935).

**Comment by jhidding** [comment]
@xyl96 Is this still relevant?

**Comment by xyl96** [comment]
yes, but this is not urgent. Basically the kinetics of chemical dissolution should also depends on temperature
